packages/hawtio/src/ui/page/HawtioPage.tsx

```diff
@@ -51,7 +58,7 @@ export const HawtioPage: React.FunctionComponent = () => {
 
   return (
     <PageContext.Provider value={{ username, plugins }}>
-      <BackgroundImage src={backgroundImages} />
+      <BackgroundImage src={backgroundImages.lg} />
```

Comment:
why the change?

Reply:
Image sources now required strings not BackgroundImageSrcMap's

Reply:
Then the problem should be that now the background image doesn't change automatically depending on the size of the display. Your change seems to force using only the lg size image for every display.

How can we adjust background image size based on the display resolution with PF 5?